docs/100.yaml

```
@@ -271,7 +271,7 @@
   title  : Fourier Series
 77:
   title  : Sum of kth powers
-  decl   : sum_range_pow
+  decl   : sum_range_pow'
```

Any reason not to include both variations here?
```suggestion
  decls  :
    - sum_range_pow
    - sum_range_pow'
```

---

no, I didn't realize that this was an option.